src/test/regress/sql/multi_behavioral_analytics_basics.sql

```diff
@@ -219,9 +210,8 @@ TRUNCATE agg_results;
 INSERT INTO agg_results(user_id)
 SELECT user_id
 FROM users_table
-WHERE (value_1 = 10
-       OR value_1 = 11
-       OR value_1 = 12)
+WHERE (value_1 >= 1
```

Comment:
Here we converted OR to AND. Again, this runs different parts of the code. Why did we change this?

Reply:
I tried to put a filter to the results. Going to add ORs back